_scss/_typography.scss

```diff
@@ -152,6 +152,7 @@ h4, .h4 {
 p, .body1 {
   font-size: 1rem;
   line-height: 1.5;
+  letter-spacing: 0.05rem;
```

Comment:
@mark-dr I'm not sure if it should be a font property or letter-spacing change? WDYT? See https://github.com/docker/docs/pull/16201#issuecomment-1332933442